pkg/api/testing/serialization_test.go

```diff
@@ -205,11 +205,7 @@ func TestCommonKindsRegistered(t *testing.T) {
 func TestRoundTripTypes(t *testing.T) {
 	seed := rand.Int63()
 	fuzzer := fuzzer.FuzzerFor(FuzzerFuncs, rand.NewSource(seed), legacyscheme.Codecs)
-
-	nonRoundTrippableTypes := map[schema.GroupVersionKind]bool{
-		{Group: "componentconfig", Version: runtime.APIVersionInternal, Kind: "KubeProxyConfiguration"}:     true,
-		{Group: "componentconfig", Version: runtime.APIVersionInternal, Kind: "KubeSchedulerConfiguration"}: true,
-	}
+	nonRoundTrippableTypes := map[schema.GroupVersionKind]bool{}
```

Comment:
do we have a roundtrip_test.go like https://github.com/kubernetes/kubernetes/blob/d3bd0eb1d5cefc25e4476d8dc086ebd90439ef4e/staging/src/k8s.io/apiserver/pkg/apis/audit/install/roundtrip_test.go#L26 for componentconfig types? If not, that would be a good follow-up.

Reply:
I added a TODO for this in fuzzer.go in the componentconfig pkg, but will be applied first when the structs are split out to different repos.